app/react/Settings/components/CollectionSettings.js

```diff
@@ -145,6 +167,43 @@ export class CollectionSettings extends Component {
               </label>
               <Control.text id="collection_name" model=".site_name" className="form-control" />
             </div>
+
```

Comment:
The fix for this duplicated code is not easy at all. I tried several alternatives and all of them (abstracting to a component, abstracting to a render function) introduce at least one race condition with the states (because of how the react-redux-form package manages it's own state). So I agreed with @daneryl that:
- Because this is markup and not algorithms, it's not that bad, to have it duplicated, in opposition to algorithms. It would even obfuscate the code.
- The whole view needs a big refactor to clean it up.

That said, we are assuming this CC issue for the time being and I'll open a Tech Debt issue to tackle the CollectionSettings refactor at some point.